spec/ParseGraphQLServer.spec.js

```diff
@@ -88,8 +88,8 @@ describe('ParseGraphQLServer', () => {
 
     it('should initialize parseGraphQLSchema with a log controller', async () => {
       const loggerAdapter = {
-        log: () => {},
-        error: () => {},
+        log: () => { },
```

Comment:
Could you remove these spaces inside the brackets? Seems to be added by your local lint?

Reply:
fixed lint errors